character-count.js

```diff
@@ -0,0 +1,17 @@
+function characterCount(arr) {
+
+    var obj = {}
+    for (var i = 0; i < arr.length; i++) {
+        var key = arr[i].length;
+        if (!obj[key]) {
```

Comment:
Be careful with this. If you have an object has a key with a value of 0, this will also be false. This doesn't really mean "if key exist" it actually means if the value at key is false. So for example:
```javascript
var a = { count: 0 }

if (a.count) { //evaluates to false.
}
```